### PR TITLE
chore: update to the latest major version of eslint

### LINKED ISSUE #12: Update to ESLint v1.x

### PR DESCRIPTION
**Pull request by remcohaszing**
Update to eslint ^1.1.0

Closes #12

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,22 @@
 extends: tlvince
 env:
   node: true
+
+rules:
+  # These rules have been modified in the update of eslint@^1.0.0.
+  # The configuration is overridden, because the upstream ruleset hasn't been upgraded to this eslint version.
```
Comment (tlvince) [inline .eslintrc]: Thanks. I should update eslint-config-tlvince, though I may switch over to standard.

File: package.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "name": "eslint-plugin-jasmine",
   "description": "ESLint rules for Jasmine",
   "main": "index.js",
-  "peerDependencies": {
-    "eslint": ">=0.8.0"
+  "dependencies": {
+    "eslint": "^1.1.0"
```
Comment (tlvince) [inline package.json]: Shouldn't this be in devDeps?

File: test/rules/no-spec-dupes.js [hunk 3]
```diff
@@ -49,8 +49,7 @@ eslintTester.addRuleTest('lib/rules/no-spec-dupes', {
 
     // 'block'
     {
-      args: [
-        2,
```
Comment (tlvince) [inline test/rules/no-spec-dupes.js]: Was the `2` left off intentionally?

File: package.json (hunk 2 above)
Comment (remcohaszing) [inline package.json]: You're right, I'll amend it.

**Comment by remcohaszing** [comment]
The patch is modeled after this plugin migrating as well: https://github.com/Gillespie59/eslint-plugin-angular/commit/10ed70c65a9bf9fd119b4c03bf709a0548103f2a

File: test/rules/no-spec-dupes.js (hunk 3 above)
Comment (remcohaszing) [inline test/rules/no-spec-dupes.js]: Yes.

**Comment by tlvince** [comment]
Looks good to me. Thanks.

**Pull request merged**